### ISSUE TITLE
[TRIGGER] Discord message

### ISSUE THREAD

**Issue by sirloinofbeef** [PipedreamHQ/pipedream#1750]
**Discord Message**
Whilst Pipedream can trigger actions on Discord, there is no simple way for Discord users to communicate with the Discord bot, triggering Pipedream workflows. 

Triggers which would assist with this include:
- Listening for direct messages from users to the bot
- and / or listening in a specified channel for a specific command string (eg: !runprocess)

**API**
https://discord.com/developers/docs/reference

**Comment by yasinbordbar** [comment]
Hi, please assign me to do this.

**Comment by dylburger** [comment]
Yes I hear you. For the event source to use webhooks, Discord would have to support a subscription webhook API that allows us to dynamically create webhooks to point to specific event sources on event source creation. As a Discord user, you could reach out with that feedback (Google, GitHub, and many other first-class APIs support that).

Otherwise I think we'll have to build a gateway service that allowed specific event sources to subscribe to those events. Since work is required on our part, I'm going to add the `blocked` label to this ticket for now.
